audio/audacity/Portfile

```diff
@@ -63,10 +64,29 @@ homepage            http://www.audacityteam.org/
 
 compiler.blacklist  {*gcc-[234].[0-8]}
 
-wxWidgets.use       wxWidgets-3.0
+if {[variant_exists wx32] && [variant_isset wx32]} {
+    wxWidgets.use   wxWidgets-3.2
+    # we want to use the wxWidgets-3.2 "bleeding edge" port!
+    wxWidgets.port  wxWidgets-3.2
```

Comment:
Doesn't it work for you without setting `wxWidgets.port` explicitly? I fixed a bug a while ago and would like to know in case I didn't fix the problem in a proper way.

Reply:
I see that this is no longer required, you fixed the glitch that made this necessary.